README.md

```diff
@@ -27,8 +27,6 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
   <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] Kindly make sure your work is documented in a professional way. As stated in the table of content in your README, Kindly consider adding all the points that apply to your project.
like:
```sh
Get started
Author
...
```
As is the case in the template you just used.